commit/core.py

```diff
@@ -611,7 +611,7 @@ def build_operator( self ) :
         print '   [ %.1f seconds ]' % ( time.time() - tic )
 
 
-    def fit( self, tol_fun = 1e-3, max_iter = 100, verbose = 1 ) :
+    def fit( self, tol_fun = 1e-3, max_iter = 100, verbose = 1, x0 = None ) :
```

Comment:
Can you implement a check that the size of the x0 array matches with the linear operator A?